templates/extras/simp.md

```diff
@@ -238,6 +238,14 @@ into `f`).
 
 ## More advanced features.
 
+Using the command [`mk_simp_attribute`](https://leanprover-community.github.io/mathlib_docs/commands.html#mk_simp_attribute),
+you can make your own `@[simp]` attribute, but with a key difference;
+lemmas tagged with `@[new_attr]` are _not_ called by default with `simp`, but
+can be called at will with `simp with new_attr`. This can often replace lengthy 
+`simp only [...]` calls, and facilitate easier-to-read code. Some examples of common usage are 
+[`mfld_simps`](https://github.com/leanprover-community/mathlib/blob/5993b90e4b3acc615a08f969c670b43275c1f7de/src/data/equiv/local_equiv.lean#L67),
+and [`field_simps`](https://github.com/leanprover-community/mathlib/blob/master/src/algebra/group_with_zero/basic.lean#L43).
```

Comment:
Our documentation generator has some handy redirect URLs (which should always end up at a permalink to a very recent commit of `master`):
```suggestion
[`mfld_simps`](https://leanprover-community.github.io/mathlib_docs/find/simp_attr.mfld_simps/src),
and [`field_simps`](https://leanprover-community.github.io/mathlib_docs/find/simp_attr.field_simps/src).
```

Otherwise LGTM.